lib/signex/helper.ex

```diff
@@ -40,4 +40,28 @@ defmodule SignEx.Helper do
   defp hash_key(key) do
     :crypto.hash(:md5, key)
   end
+
+  def fetch_keys(collection, keys) do
```

Comment:
I would disagree on the name as filters should not normally fail. 
The above code looks a little simpler but I think it is worth returning which keys were missing.